library/rust

```diff
@@ -1,14 +1,24 @@
-# this file is generated via https://github.com/rust-lang-nursery/docker-rust/blob/28f9bc3266a89d640a23d29a5056cc66fb6e38d3/generate-stackbrew-library.sh
+# this file is generated via https://github.com/rust-lang-nursery/docker-rust/blob/18e7c2bdae94a693bcad096af6033ec8a77122a2/generate-stackbrew-library.sh
 
 Maintainers: Steven Fackler <sfackler@gmail.com> (@sfackler)
 GitRepo: https://github.com/rust-lang-nursery/docker-rust.git
 
-Tags: 1.19.0-stretch, 1-stretch, 1.19-stretch, stretch, 1.19.0, 1, 1.19, latest
+Tags: 1.20.0-stretch, 1-stretch, 1.20-stretch, stretch, 1.20.0, 1, 1.20, latest
+Architectures: amd64, arm32v7, arm64v8, i386
+GitCommit: bf30ee63bff0a381ccacdb86436bb7da5dc9c853
+Directory: 1.20.0/stretch
+
+Tags: 1.20.0-jessie, 1-jessie, 1.20-jessie, jessie
+Architectures: amd64, arm32v7, arm64v8, i386
+GitCommit: bf30ee63bff0a381ccacdb86436bb7da5dc9c853
+Directory: 1.20.0/jessie
+
+Tags: 1.19.0-stretch, 1.19-stretch, 1.19.0, 1.19
```

Comment:
Any particular reason to keep the 1.19 versions listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
Ah sure I'll remove them.

Reply:
If there's a chance of 1.19.1 being released (hence, 1.19 still being supported), we don't mind if you keep them, but usually this is just maintainers not being aware of this part of the process. :innocent:

Reply:
That could happen but it'd be rare - we've only cut a single point release (1.15.1) in the last couple of years. Seems like we can always add a 1.19.1 image set in later if necessary.

Reply:
Indeed! :+1: :heart: